### DISCUSSION TITLE
Neovim too slow

### DISCUSSION THREAD

**Discussion by CLab001** [kovidgoyal/kitty#5028] [General]
Hello! 

My Neovim is slow, the snippets and auto suggestions are slower than Konsole - litterally sluggish while Konsole is super fast, instant results, now slowdown. I love kitty, but I cant use it like that. Th
[nvim-config.zip](https://github.com/kovidgoyal/kitty/files/8584634/nvim-config.zip)
anks for reading and I hope you have suggestions for me.

I have added my neovim config to the attachments, you can check it out. 
I have problem with Neovim using the following plugins:

require "user.options"
require "user.keymaps"
require "user.plugins"
require "user.cmp"
require "user.colorscheme"
require "user.lsp"
require "user.telescope"
require "user.treesitter"
require "user.autopairs"
require "user.comment"
require "user.gitsigns"
require "user.nvim-tree"
require "user.bufferline"
require "user.lualine"
require "user.toggleterm"
require "user.project"
require "user.impatient"
--require "user.indentline"
require "user.alpha"
require "user.whichkey"
require "user.autocommands"
vim.o.guicursor = 'n-v-c-sm-i-ci-ve:block,r-cr-o:hor20,a:blinkwait0-blinkoff0-blinkon0-Cursor/lCursor'
[nvim-config.zip](https://github.com/kovidgoyal/kitty/files/8584628/nvim-config.zip)

**Comment by kovidgoyal** [comment]
There is no way in hell kitty is slower than Konsole. Konsole is by far one of the slowest terminal emulators out there. Just google for terminal benchmarks and see for yourself. If you are seeing inferior performance it will be because neovim/one of the plugins is doing something different, possibly based on TERM or some other heuristics it uses. I dont use neovim so I cant help you with that.

**Reply by CLab001** [reply]
Ok. Cool. I need to find out whats causing that. I like Kitty-cat a alot.

**Comment by CLab001** [comment]
Actyally it would be super cool if someone would look into this. I am no expoert on neither vim/neovim or kitty terminal. 

This matter becaue VIM is a super popular editor and people are likely going to use similar level of config/plugins with their vim.
I am going to use another terminal emulator for now but I would love to use Kitty in the future. I am a fan otherwise.

**Reply by MrRoiz** [reply]
Are you still facing the same issue with kitty? I'm using neovim on kitty and sometimes it just takes a lot of time to show auto completes and all those things. I'm not sure if kitty is the issue, I haven't tried with other term emulators. But I found a thread on Reddit that says that Kitty was pretty slow with neovim but I don't know if that's true or not.
